src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -218,23 +253,33 @@ private Card createJobCompletedCard() {
                 status = result.toString();
                 summary += " " + status;
             }
-
             statusFact.setValue(status);
         } else {
             statusFact.setValue(" Completed");
             summary += " Completed";
         }
 
         factsBuilder.addRemarks();
-        addScmDetails();
-
-        String activityTitle = "Update from " + jobName + ".";
-        String activitySubtitle = "Latest status of build #" + run.getNumber();
-        Section section = new Section(activityTitle, activitySubtitle, factsBuilder.collect());
-
+        processScmDetails(true);
+
+        String activityTitle = "";
+        String activitySubtitle = "";
+        Section section = null;
+
+        if (isCompactNotification()) {
```

Comment:
Perhaps a good time to split the card definition into a subclass instead to support different card styles.

Reply:
@casz This is a good suggestion, will add this to my todo list